blobs/howto/python/blob-devguide-py/blob-devguide-blobs.py

```diff
@@ -0,0 +1,410 @@
+import io
+import os, uuid
+import random
+import time
+from azure.core.exceptions import HttpResponseError, ResourceExistsError
+from azure.identity import DefaultAzureCredential
+from azure.storage.blob import BlobServiceClient, BlobClient, ContainerClient, BlobLeaseClient, BlobPrefix, ContentSettings
+
+class BlobSamples(object):
+
+    # <Snippet_upload_blob_data>
+    def upload_blob_data(self, blob_service_client: BlobServiceClient, container_name):
+        blob_client = blob_service_client.get_blob_client(container=container_name, blob="sample-blob.txt")
+        data = "Sample data for blob"
```

Comment:
Could you change this to `data = b"Sample data for blob"` (including the `b` for binary)? We tend to prefer binary data over strings. If this is left as a string, it will be automatically encoded UTF-8 which is not always obvious to the customer.